src/booking/book.messages.js

```diff
@@ -47,3 +47,15 @@ module.exports.removeRoomsMessage = (event) => {
     })
     .buildMessage();
 };
+
+module.exports.removeRoomMessage = (event, roomEmail) => {
```

Comment:
```suggestion
module.exports.removeRoomMessage = (event, roomName) => {
```

Please change that to `roomName` as you are passing the room name here not its email